enhancements/console/customize-catalogs.md

```diff
@@ -0,0 +1,285 @@
+---
+title: customize-catalogs
+authors:
+  - "@lokanandaprabhu"
+reviewers:
+  - "@jerolimov"
+  - "@spadgett"
+approvers:
+  - "@spadgett"
+  - "@cvogt"
+api-approvers:
+  - "@JoelSpeed"
+creation-date: 2022-07-27
+last-updated: 2022-09-14
+tracking-link: 
+  - https://issues.redhat.com/browse/ODC-6695
+status: implementable
+---
+
+# Customize Catalogs
+
+## Summary
+
+The OpenShift Console provides a "Developer Catalog" which enables 
+the user to create apps and services based on Operator backed services, helm charts, templates, etc.
+
+Cluster administrators can provide additional catalog types (providers) 
```

Comment:
I know this suggestion was done earlier from me, but I understand later that types ~= Sub-catalogs and types and not the same then providers in our frontend code. Multiple providers can add items to one type. So we should remove this here to don't confuse people later.

```suggestion
Cluster administrators can provide additional catalog types
```